spec/02-integration/03-dao/03-crud_spec.lua

```diff
@@ -402,7 +407,15 @@ helpers.for_each_dao(function(kong_config)
         assert.equal(1, #rows)
         assert.same(first_api, rows[1])
       end)
-      it("filter supports a boolean value", function()
+      it("filter supports a truthy boolean value", function()
+        local rows, err, _ = apis:find_page {
+          name = "fixture_5",
+          https_only = "true"
+        }
+        assert.falsy(err)
+        assert.is_table(rows)
```

Comment:
Now that we test both values in boolean attributes, and to avoid the below test to produce a false positive (returning values with `https_only=true`, we really should update both of these to ensure that all fetched rows indeed have `https_only` set to `true` or `false`, accordingly.

Reply:
I support this idea, that was the reason for ensuring only one row has a true value. Ensuring the values would be a great step further.